app/graphql/types/fees/object.rb

```diff
@@ -6,6 +6,7 @@ class Object < Types::BaseObject
       graphql_name 'Fee'
       implements Types::Invoices::InvoiceItem
 
+      field :add_on, Types::AddOns::Object, null: true
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `add_on` should appear before `vat_rate`.